src/react/components/connect.tsx

```diff
@@ -1,9 +1,13 @@
+/* tslint:enable: allowSyntheticDefaultImports */
+
 import * as React from "react";
 
 import shallowEqual from "../../utils/shallowEqual";
 import propValidation from "../../utils/propsValidation";
 import bindActions from "../../utils/bindActions";
 
+const hoistNonReactStatics = require("hoist-non-react-statics");
```

Comment:
sadly typescript can't use this module with the `import` statement. :(

Reply:
i've tried it... i don't know sufficient TS to know why this don't works.

On my JS projects I use just `import hoistNonReactStatics from 'hoist-non-react-statics';`